website/docs/guides/internals.md

```diff
@@ -65,60 +63,63 @@ The typical content of `.scala` looks like
             └── test.scala
 ```
 
-In particular, `.scala/.bloop` contains Bloop project files and Bloop's own working directories, and
-`.scala/project_*` contains byte code and TASTy files, and generated sources.
+In particular, `.scala/.bloop` contains Bloop project files and Bloop's own working directories, and `.scala/project_*` contains byte code, TASTy files, and generated sources.
 
 ## Preprocessing
 
-Some sources that Scala CLI accepts cannot be passed as is to scalac / dotty. That's the case for
-- `.sc` files, that can contain top-level definitions not accepted by scalac or dotty
-- `.scala` files with non-commented `using` directives
-- `.sc` or `.scala` files with `import $ivy`s.
+Some source code files that Scala CLI accepts cannot be passed as-is to `scalac`.
+This is the case for:
+- `.sc` files, which can contain top-level definitions not accepted by `scalac`
+- `.scala` files that have uncommented `using` directives
+- `.sc` or `.scala` files with `import $ivy` statements
 
 In all of those cases, Scala CLI parses the top of those files, and looks for `import $ivy` or `using` directives.
-It then replaces those either by dummy imports (for the `import $ivy`s) or space characters
-(for non-commented `using` directives).
+It then replaces those with either dummy imports (for the `import $ivy` statements) or space characters (for non-commented `using` directives).
```

Comment:
@alvinj those dummy imports sometimes are causing warnings, why we cannot replace them with spaces, same as for using directves?